pom.xml

```diff
@@ -738,6 +765,20 @@
 
       <plugin>
         <artifactId>maven-failsafe-plugin</artifactId>
+        <version>3.0.0-M3</version>
```

Comment:
Shall we move the version thing to pluginManagement instead? The same goes for surefire plugin.

Reply:
👍